Simplenote/SPFallbackTextContainer.swift

```diff
@@ -0,0 +1,19 @@
+class SPFallbackTextContainer: NSTextContainer {
```

Comment:
Annoying comment: Can we add either an `UIKit` import or something, so that the class declaration isn't the very first one in the file?